SpatialGDK/Source/SpatialGDK/Public/Tests/TestActor.h

```diff
@@ -12,7 +12,7 @@ class ATestActor : public AActor
 {
 	GENERATED_BODY()
 
-	ATestActor() { SetReplicates(true); }
+	ATestActor() { bReplicates = true; }
```

Comment:
How come this change? SetReplicates appears to do only modify `bReplicates` if `GetLocalRole() == ROLE_Authority`

Reply:
https://github.com/improbableio/UnrealEngine/blob/experiment/4.26/Engine/Source/Runtime/Engine/Private/Actor.cpp#L3466